Comments from Guy: May-12 (#2)
ietf-rats/draft-voit-rats-attestation-results: draft-voit-rats-attestation-results.md

It is upon this root of trust that unique, non-repudiable identities may be founded.
Example attested identities may include:

* a type of hardware chip used for the Attesting Environment (e.g., TPM2.0)

> * a unique instance of a running Attesting Environment (e.g., LDevID {{802.1AR}}, Implementation ID {{-PSA}})

Reviewer: ```suggestion
* a unique instance of a running Attesting Environment (e.g., LDevID {{802.1AR}}, Instance ID {{-PSA}})
```

Author: I initially placed Instance ID there.  But further reading led me to believe that multiple Instance ID's could be loaded into one TrustZone.  Could you confirm it is 1:1 between Attesting Environment and Instance ID?    I actually expect we are going to need to build more explicit mappings of identities in this section (or the Appendices) over time.

Reviewer: Implementation ID is a kind of "class identifier" that is common to all devices sharing the same PSA RoT implementation.  Instance ID is the unique identifier of a PSA RoT instance.

### Attester

Per {{I-D.ietf-rats-architecture}} Figure 2, an Attester and a corresponding Attesting Environment might not share common boundaries.

> An example of such a case might be a laptop which records its security measurements within a TPM.
> In such cases, where connections are being established directly to an Attester but not to the Attesting Environment, the Verifier must include sufficient information in the Attestation Results to enable the Relying Party to have confidence that the Attester's trustworthiness is represented by Trustworthiness Claims signed by the appropriate Attesting Environment.

Reviewer 2: ```suggestion
One example of such a case is a laptop that records its security measurements via the protected capabilities of a TPM.
In cases where connections are being established directly to an Attester but not to one of its Attesting Environments, a Verifier must include sufficient information in corresponding Attestation Results to enable Relying Parties to have confidence that the Attester's trustworthiness is represented via Trustworthiness Claims signed by the appropriate Attesting Environment.
```

The mechanism for communicating the Attesting Environment identity (and if it is different, the Attester identity <!--Henk(old): there is a deterministic relationship here, see AuthSecID in interaction models -->) may be either implicit or explicit within an instance of Attestation Results.
An example of explicit communication would be to include the following Identity Evidence directly in the Attestation Results: a unique identifier for an Attesting Environment, the name of a key which can be provably associated with that unique identifier, and the set of Attestation Results are signed using that key.

> An example of implicit communication would be to include the following Identity Evidence: a signature which has been made across the Verifier's Attestation Results.

Reviewer 2: ```suggestion
An example of implicit communication would be to include the following Identity Evidence: a signature over all Attestation Results generated by a Verifier.
```

     3. Disallow any information exchange into a Relying Party context for which that Verifier B appraised Trustworthiness Vector is not qualified.

As link layer protocols re-authenticate, steps (1) to (2) and steps (3) to (6) will independently refresh.
This allows the Trustworthiness of Attester to be continuously re-appraised.
There are only specific triggers which will refresh Evidence generation (1), Attestation Result generation (2), and in consequence AR-augmented Evidence generation (4):

> * live-cycle events, e.g. a change to an Authentication Secret of the Attester or an update of a software component

Reviewer: non-native question: wouldn't this be life-cycle instead?

Author: Yes
